### PR TITLE
(DOCSP-26617): Write Custom User Data from Function

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Light documentation on how you can write custom user data from Atlas Function. 

Adding b/c there won't be MongoDB Data Access for GA, but should be documented way to write to custom user data. 

### Jira

- https://jira.mongodb.org/browse/DOCSP-26617

### Staged Changes

- [Custom User Data](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-26617/sdk/flutter/users/custom-user-data)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/flutter/users/custom-user-data.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@ Before You Begin
 ----------------
 
 To use custom user data, you must first enable custom user data
-on the App Services:
+in App Services:
```
Comment (dacharyc) [inline source/sdk/flutter/users/custom-user-data.txt:32]: I can't comment directly but ln 34 below should be "Create an App"

**Review by dacharyc — APPROVED** [review]
Nice - this is an excellent suggestion for doing this while the SDK does not yet have MongoDB Data Access. LGTM!

I did spot one typo on the page unrelated to your PR that you might want to fix while you're in here. I've seen a handful of these lately - may be related to the mass naming update we did with docdoctor a while ago ago.

File: source/sdk/flutter/users/custom-user-data.txt (hunk 1 above)
Comment (mongodben) [reply source/sdk/flutter/users/custom-user-data.txt:32]: will fix typo in this PR while we in here!

File: source/sdk/flutter/users/custom-user-data.txt [hunk 2]
```diff
@@ -29,7 +29,7 @@ Before You Begin
...
 
 #. :ref:`Create a App <create-a-realm-app>`.
```
Comment (mongodben) [inline source/sdk/flutter/users/custom-user-data.txt:34]:
```suggestion
#. :ref:`Create an App <create-a-realm-app>`.
```

**Pull request merged**